source/standards/reliability-engineering.html.md.erb

```diff
@@ -0,0 +1,23 @@
+---
+title: Reliability Engineering
+expires: 2018-12-01
+---
+
+# <%= current_page.data.title %>
+
+Reliability Engineering provide a shared platform to GDS teams made up of tools and services to set up and maintain a service, for example by:
```

Comment:
The wording's a little ambiguous about who/what's made up of tools and services, and it may be possible to avoid using the word "service" twice. 

I'd recommend changing this lead-in line to something along the lines of:

> Reliability Engineering provide a shared platform to distribute tools and services. This includes support for: